src/cascadia/WinRTUtils/inc/WtExeUtils.h

```diff
@@ -0,0 +1,93 @@
+
+static constexpr std::wstring_view WtExe{ L"wt.exe" };
+static constexpr std::wstring_view WtdExe{ L"wtd.exe" };
+static constexpr std::wstring_view WindowsTerminalExe{ L"WindowsTerminal.exe" };
+
+static constexpr std::wstring_view LocalAppDataAppsPath{ L"%LOCALAPPDATA%\\Microsoft\\WindowsApps\\" };
+
+// Function Description:
+// - This is a helper to determine if we're running as a part of the Dev Build
+//   Package or the release package. We'll need to return different text, icons,
+//   and use different commandlines depending on which one the user requested.
+// - Uses a C++11 "magic static" to make sure this is only computed once.
+// - If we can't determine if it's the dev build or not, we'll default to true
+// Arguments:
+// - <none>
+// Return Value:
+// - true if we believe this extension is being run in the dev build package.
+bool IsDevBuild()

```

Comment:
we can't do this. Including bare functions in a header is a way to get hurt during linking if two files in the same project `#include` it.